docs/resources/task.md

```diff
@@ -281,25 +283,86 @@ spec:
 
 ## Permission Set
 
+### Generic
+
 A set of "permissions" assigned to the `task` instance, format depends on the cloud provider
 
-#### Amazon Web Services
+### Cloud-specific
+
+#### Kubernetes
+
+The name of a service account in the current namespace.
+
+### Amazon Web Services
 
 An [instance profile `arn`](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html), e.g.:
 `permission_set = "arn:aws:iam:1234567890:instance-profile/rolename"`
 
-#### Google Cloud Platform
+### Google Cloud Platform
 
 A service account email and a [list of scopes](https://cloud.google.com/sdk/gcloud/reference/alpha/compute/instances/set-scopes#--scopes), e.g.:
 `permission_set = "sa-name@project_id.iam.gserviceaccount.com,scopes=storage-rw"`
 
-#### Microsoft Azure
+### Microsoft Azure
```

Comment:
I don't get this title reworking. There are no generic options for `permission_set`, so there shouldn't be a `### Generic` section?